eve/main.yml

```diff
@@ -2537,12 +2537,6 @@ stages:
           name: Import Solution next example-solution archive
           env:
             SOLUTION_ARCHIVE: /var/tmp/example-solution-next.iso
-      - ShellCommand:
-          <<: *bastion_ui_tests
-          name: Run Solutions UI tests
-          env:
-            <<: *_env_bastion_ui_tests
-            TEST_FILTER: "e2esolutions"
       - ShellCommand: *collect_cypress_result_ssh
       - Upload: *upload_cypress_artifacts
```

Comment:
I think you want to remove those two as well since we do no longer run any cypress tests

Reply:
Thanks @TeddyAndrieux! 
Updated in https://github.com/scality/metalk8s/pull/3167/commits/84506135f63f9b7ff12e07de31a8a4d6c41bc856